src/Samples/Toolkit.SampleApp.Maui/Samples/FeatureFormViewSample.xaml.cs

```diff
@@ -37,16 +37,14 @@ private async void mapView_GeoViewTapped(object sender, Esri.ArcGISRuntime.Maui.
             }
         }
 
-        private ArcGISFeature? GetFeature(IEnumerable<IdentifyLayerResult> results, out FeatureFormDefinition? def)
+        private ArcGISFeature? GetFeature(IEnumerable<IdentifyLayerResult> results)
         {
-            def = null;
             if (results == null)
                 return null;
             foreach (var result in results.Where(r => r.LayerContent is FeatureLayer layer && (layer.FeatureFormDefinition is not null || (layer.FeatureTable as ArcGISFeatureTable)?.FeatureFormDefinition is not null)))
             {
                 var feature = result.GeoElements?.OfType<ArcGISFeature>()?.FirstOrDefault();
-                def = (result.LayerContent as FeatureLayer)?.FeatureFormDefinition ?? ((result.LayerContent as FeatureLayer)?.FeatureTable as ArcGISFeatureTable)?.FeatureFormDefinition;
-                if (feature != null && def != null)
+                if (feature != null)
                 {
                     return feature;
                 }
```

Comment:
This method falls back to returning null anyway, so you might as well `return feature` without a null check